<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>1098</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGraphicsView" name="gvMain">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <property name="backgroundBrush">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvSecond">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>100</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <property name="backgroundBrush">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Play"/>
    <addaction name="action_Stop"/>
    <addaction name="action_Save"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionReset"/>
    <addaction name="actionWindow_Switch"/>
   </widget>
   <widget class="QMenu" name="menu_Set">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="action_Setting"/>
   </widget>
   <widget class="QMenu" name="menu_Language">
    <property name="title">
     <string>&amp;Language</string>
    </property>
    <addaction name="actionEnglish_Chinese"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Image">
    <property name="title">
     <string>&amp;Image</string>
    </property>
    <addaction name="actionHistogram"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Set"/>
   <addaction name="menu_Image"/>
   <addaction name="menu_Language"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Play"/>
   <addaction name="action_Stop"/>
   <addaction name="action_Save"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionReset"/>
   <addaction name="action_Setting"/>
   <addaction name="actionWindow_Switch"/>
  </widget>
  <action name="action_Play">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/play</normaloff>:/icon/play</iconset>
   </property>
   <property name="text">
    <string>&amp;Play</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/pause</normaloff>:/icon/pause</iconset>
   </property>
   <property name="text">
    <string>Pa&amp;use</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/save</normaloff>:/icon/save</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset>
     <normalon>:/icon/zoomin</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/zoomout</normaloff>:/icon/zoomout</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/view-refresh</normaloff>:/icon/view-refresh</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="action_Setting">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/setting</normaloff>:/icon/setting</iconset>
   </property>
   <property name="text">
    <string>&amp;Setting</string>
   </property>
  </action>
  <action name="actionEnglish_Chinese">
   <property name="text">
    <string>English&lt;-&gt;Chinese</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About </string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
  <action name="actionWindow_Switch">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/window</normaloff>:/icon/window</iconset>
   </property>
   <property name="text">
    <string>Window Switch</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
